addons/web/static/src/core/domain_selector/domain_selector_fields.js

```diff
@@ -338,6 +348,7 @@ export const FIELD_DESCRIPTIONS = {
     float: NUMBER,
     html: TEXT,
     integer: NUMBER,
+    json: JSON_FIELD,
```

Comment:
Hello @Polymorphe57 For the fw-port to master, do you want a specific name for this or is it good enough for you? 
(I can't do `json: JSON`, as it shadows the JSON class usage, so I named it JSON_FIELD, and didn't change the `json`)